www-apps/radicale/radicale-2.1.8.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6} )
+
+inherit eutils distutils-r1 user
+
+MY_PN="Radicale"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A simple CalDAV calendar server"
+HOMEPAGE="http://radicale.org/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+bcrypt"
+
+RDEPEND=">=dev-python/vobject-0.9.5[${PYTHON_USEDEP}]
+	bcrypt? ( dev-python/passlib[bcrypt,${PYTHON_USEDEP}] )"
+
+S=${WORKDIR}/${MY_P}
+
+RDIR=/var/lib/radicale
+LDIR=/var/log/radicale
+
+pkg_pretend() {
+	if [[ -f ${RDIR}/.props && ${MERGE_TYPE} != buildonly ]]; then
+		eerror "It looks like you have a version 1 database in ${RDIR}."
+		eerror "You must convert this database to version 2 format before upgrading."
+		eerror "You may want to back up the old database before migrating."
+		eerror "Please see the migration guide for details:"
+		eerror "  http://radicale.org/1to2/"
+		eerror "After conversion, please delete the old database (including hidden .props files),"
+		eerror "then copy the exported database into ${RDIR}."
+		eerror "You should finish with a ${RDIR}/collection-root directory."
+		die
+	fi
+}
+
+pkg_setup() {
+	enewgroup radicale
+	enewuser radicale -1 -1 ${RDIR} radicale
+}
+
+python_install_all() {
+	rm README* || die
+
+	# init file
+	newinitd "${FILESDIR}"/radicale.init.d radicale
+
+	# directories
+	diropts -m0750 -oradicale -gradicale
+	keepdir ${RDIR}
+	diropts -m0755 -oradicale -gradicale
+	keepdir ${LDIR}
+
+	# config file
+	insinto /etc/${PN}
+	doins config logging
+
+	# fcgi and wsgi files
+	exeinto /usr/share/${PN}
+	doexe radicale.wsgi
+	doexe radicale.fcgi
```

Comment:
Please combine this into one `doexe` with two files. Otherwise, people are going to repeatedly wtf before they notice that's fcgi/wsgi.

Reply:
Fixed in 32c014f81dfd.